src/Commands/InstallCommand.php

```diff
@@ -62,43 +66,45 @@ protected function resolveBaseStructures(): array
     }
 
     protected function resolveInfrastructurePath(): string
     {
-        return Layer::infrastructure->resolvePath(suffix: '/Laravel/Providers');
+        return Layer::infrastructure->resolvePath(prefix: $this->resolveDomainArgument(), suffix: '/Laravel/Providers');
     }
 
     protected function resolveNamespace(): string
     {
-        return Layer::infrastructure->resolveNamespace(suffix: '\\Laravel\\Providers');
+        return Layer::infrastructure->resolveNamespace(prefix: $this->resolveDomainArgument() . '\\', suffix: '\\Laravel\\Providers');
     }
 
-    protected function resolveRefactor(): void
-    {
-        if (! $this->option('skip-refactor')) {
-            $filesystem = new Filesystem;
-            $filesystem->ensureDirectoryExists($this->resolveInfrastructurePath());
-            $filesystem->moveDirectory(from: app_path(path: 'Providers'), to: $this->resolveInfrastructurePath());
-            $configPath = base_path(path: '/config/app.php');
-            $contents = Str::replace(
-                search: 'App\\Providers',
-                replace: $this->resolveNamespace(),
-                subject: $filesystem->get($configPath)
-            );
-
-            $filesystem->put(path: $configPath, contents: $contents);
-
-            foreach ($filesystem->files($this->resolveInfrastructurePath()) as $file) {
-                $contents = Str::replace(
-                    search: 'App\\Providers',
-                    replace: $this->resolveNamespace(),
-                    subject: $filesystem->get($file)
-                );
-
-                $filesystem->put(path: $file, contents: $contents);
-            }
-        }
-    }
+    // protected function resolveRefactor(): void
+    // {
+    //     if (! $this->option('skip-refactor')) {
+    //         $filesystem = new Filesystem;
+    //         $filesystem->ensureDirectoryExists($this->resolveInfrastructurePath());
+    //         $filesystem->moveDirectory(from: app_path(path: 'Providers'), to: $this->resolveInfrastructurePath());
+    //         $configPath = base_path(path: '/config/app.php');
+    //         /** @var string $contents */
+    //         $contents = Str::replace(
+    //             search: 'App\\Providers',
+    //             replace: $this->resolveNamespace(),
+    //             subject: $filesystem->get($configPath)
+    //         );
+    //
+    //         $filesystem->put(path: $configPath, contents: $contents);
+    //
+    //         foreach ($filesystem->files($this->resolveInfrastructurePath()) as $file) {
+    //             /** @var string $contents */
+    //             $contents = Str::replace(
+    //                 search: 'App\\Providers',
+    //                 replace: $this->resolveNamespace(),
+    //                 subject: $filesystem->get($file)
+    //             );
+    //
+    //             $filesystem->put(path: $file, contents: $contents);
+    //         }
+    //     }
+    // }
```

Comment:
Please put an commen why we commented it

Reply:
Okay